librz/core/cmd/cmd_analysis.c

```diff
@@ -3602,121 +3486,101 @@ static void cmd_analysis_graph(RzCore *core, const char *input) {
 	}
 }
 
-RZ_API int rz_core_analysis_refs(RzCore *core, const char *input) {
-	int cfg_debug = rz_config_get_b(core->config, "cfg.debug");
-	ut64 from, to;
-	RzOutputMode out_mode;
-	PJ *pj = NULL;
-	if (*input == '?') {
-		rz_core_cmd_help(core, help_msg_aar);
-		return 0;
-	}
-	switch (*input) {
-	default:
-		out_mode = RZ_OUTPUT_MODE_STANDARD;
-		break;
-	case 'j':
-		out_mode = RZ_OUTPUT_MODE_JSON;
-		break;
-	case '*':
-		out_mode = RZ_OUTPUT_MODE_RIZIN;
-		break;
+static bool core_search_for_xrefs_in_boundaries(RzCore *core, ut64 from, ut64 to, RzCmdStateOutput *state) {
+	if (from == UT64_MAX && to == UT64_MAX) {
+		return false;
+	} else if (!from && !to) {
```

Comment:
You can remove `else` here and just use sequence of `if`s.

Reply:
why?

Reply:
Because of `return`